docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -140,9 +135,9 @@ $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 
 [NOTE]
 ====
-The downloaded `docker-machine-driver-xhyve` binaries are compiled against a specific
-version of OS X. It is possible that the driver fails to work after an OS X version
-upgrade. In this case you can try to compile the driver from source:
+The downloaded *docker-machine-driver-xhyve* binaries are compiled against a specific version of OS X.
```

Comment:
Ugh, I am not impressed with this capitalization, but I don't work at Apple so not much I can do to protest ;) will search and replace.